crates/bevy_pbr/src/render/shadows.wgsl

```diff
@@ -1,5 +1,7 @@
 #define_import_path bevy_pbr::shadows
 
+var<private> flip_z: vec3<f32> = vec3<f32>(1.0, 1.0, -1.0);
```

Comment:
```suggestion
const flip_z: vec3<f32> = vec3<f32>(1.0, 1.0, -1.0);
```

Reply:
I should have looked that up. I was checking WGSL const expressions yesterday (compile-time evaluable expressions) and when looking at the var<private> this morning thought ‘yup, that’s the only way at the moment’. Good catch.